Code/Tools/FBuild/FBuildCore/Graph/CompilerNode.cpp

```diff
@@ -369,7 +369,12 @@ CompilerNode::~CompilerNode()
 //------------------------------------------------------------------------------
 /*virtual*/ Node::BuildResult CompilerNode::DoBuild( Job * /*job*/ )
 {
-    if ( !m_Manifest.DoBuild( m_StaticDependencies ) )
+    bool skipHashing = false;
+    if ( !m_AllowCaching && !m_AllowDistribution )
+    {
+      skipHashing = true;
+    }
```

Comment:
This is one of the bits failing clang format (bad indents).

Could all be one line as well:
```
const bool skipHashing = ( !m_AllowCaching && !m_AllowDistribution );
```